tests/unit/loadDojoConfig.ts

```diff
@@ -0,0 +1,18 @@
+import { describe, it } from 'intern!bdd';
+import * as assert from 'intern/chai!assert';
+import loadDojoConfig from '../../src/loadDojoConfig';
+
+describe('loadDojoConfig', () => {
+	it('should load the specified module if it exists', () => {
+		const config = loadDojoConfig('tests/support/mocks/dojo.config.js');
```

Comment:
will this only contain build configuration? i'm keen to avoid N amount of configs in the root of a project, I was hoping we could have a `.dojorc` that would potentially contain configuration for cli commands in general.